loop/models.py

```diff
@@ -88,7 +90,7 @@ class Block(LoopModel):
     block_name = models.CharField(max_length=50)
     district = models.ForeignKey(District)
     is_visible = models.BooleanField(default=True)
-    block_name_en = models.CharField(max_length=100, null=True)
+    block_name_en = models.CharField(max_length=100)
```

Comment:
Block post save to be written adn write corresponding log handling

Reply:
resolved